package.json

```diff
@@ -78,11 +76,12 @@
     "node-fetch": "^1.6.3",
     "postcss-loader": "^1.1.1",
     "progress": "^1.1.8",
-    "react-dom": "^15.4.0",
+    "react-dom": "^15.4.2",
     "react-helmet": "^4.0.0",
+    "react-hot-loader": "^3.0.0-beta.6",
     "react-redux": "^5.0.0",
-    "react-router": "^2.8.1",
-    "react-router-redux": "^4.0.5",
+    "react-resolver": "^3.1.0",
+    "react-router-dom": "^4.0.0-beta.5",
```

Comment:
I don't feel confortable using a beta version of react-router if we want to release a 1.0 in vitamin
beta versions can have breaking changes